services/books.ts

```diff
@@ -0,0 +1,70 @@
+import { fetchApi } from "./base";
+
+export const getAvailableBooks = async () => {
+  const url = '/v1/books/discover'
+  const response = await fetchApi(url, 'GET')
+  return response
+}
+
+export const getFinishedBooks = async () => {
+  const url = '/v1/books/finished_books'
+  const response = await fetchApi(url, 'GET')
+  return response
+}
+
+export const getReadingList = async () => {
+  const url = '/v1/books/reading_list'
+  const response = await fetchApi(url, 'GET')
+  return response
+}
+
+export const AddToReadingList = async (bookId: number) => {
+  const payload = {
+    id: bookId
+  }
+  const url = '/v1/books/add_reading_list'
+
+  const response = await fetchApi(
+    url,
+    'POST',
+    {
+      variables: payload,
```

Comment:
I had a comment about this - https://github.com/karlxavier/bookshelf-client/pull/2#discussion_r975999042

Reply:
Why is the payload encapsulated in a `variables` key?